cpp/examples/ide_secir_graph.cpp

```diff
@@ -0,0 +1,96 @@
+/* 
+* Copyright (C) 2020-2025 MEmilio
+*
+* Authors: Anna Wendler
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "ide_secir/infection_state.h"
+#include "ide_secir/model.h"
+#include "ide_secir/simulation.h"
+#include "memilio/config.h"
+#include "memilio/mobility/graph.h"
+#include "memilio/mobility/metapopulation_mobility_instant.h"
+
+int main()
+{
```

Comment:
Could you add a small description to the start of main, what this program does, and why? Something like "This is an example for running the IDE model in a graph. It demonstrates a simple setup with two nodes, no edges, for a graph without mobility".

The other comments are okay, but could be a bit more guiding (as in, not only telling what is done, but why we (need to) do it).